MusicTaste/MusicTaste/APIManagers/SpotifyAPIManager.m

```diff
@@ -146,6 +148,28 @@ - (NSDictionary *) convertSpotifyTracks:(NSArray *)tracks{
     //Returns all the data into a dictionary
     return @{@"albums": albumSet, @"artists": artistSet, @"tracks":trackSet};
 }
--(void) saveData:(NSDictionary *) spotifyData{
+-(void) saveSpotifyData:(NSDictionary *) spotifyData{
+    //Saves all Spotify Data such as there top genres,tracks,artist,album,and the artist images to the Parse Database
+    PFUser *current = [PFUser currentUser];
+    current[@"genres"] = [NSArray arrayWithArray:spotifyData[@"genres"]];
+    current[@"tracks"] = [NSArray arrayWithArray:spotifyData[@"tracks"]];
+    current[@"artists"] = [NSArray arrayWithArray:spotifyData[@"artists"]];
+    current[@"albums"] = [NSArray arrayWithArray:spotifyData[@"albums"]];
+    current[@"images"] = [NSArray arrayWithArray:spotifyData[@"images"]];
+    [PFUser.currentUser saveInBackground];
 }
+-(void) saveSpotifyUserData:(NSDictionary *) spotifyData{
```

Comment:
SaveSpotifyUserData that stores the Profile Picture and Name to the User File for future user

Reply:
I see that you are retrieving the data from spotify and assigning it to the model class, but where/how does it save? Generally we use the word "save" to refer to when we are storing something in a file/database or similar (like you are with Parse above).

Reply:
In this case, I'm storing it locally within User.h. Do you have alternatives naming if this is the case?

Reply:
Generally we use set, e.g. setSpotifyUserData. Another one could be assign